assigner/config.py

```diff
@@ -136,10 +136,8 @@ def __getattr__(self, key):
         # Keys contained dashes can be called using an underscore
         key = key.replace("_", "-")
 
-        try:
-            return self.data[key]
-        except KeyError as e:
-            if key == "roster":
-                self.data[key] = []
-                return self.data[key]
-            raise e
```

Comment:
This _is_ more Pythonic. Without reading into it more, why are we nuking it?

Reply:
This check fails to catch the `self.data['roster'] is None` case.